src/planner/binder/statement/bind_create.cpp

```diff
@@ -211,6 +212,33 @@ BoundStatement Binder::Bind(CreateStatement &stmt) {
 		break;
 	}
 	case CatalogType::TABLE_ENTRY: {
+		// If there is a foreign key constraint, resolve primary key column's index from primary key column's name
+		auto &create_info = (CreateTableInfo &)*stmt.info;
+		auto &catalog = Catalog::GetCatalog(context);
+		for (idx_t i = 0; i < create_info.constraints.size(); i++) {
+			auto &cond = create_info.constraints[i];
+			if (cond->type == ConstraintType::FOREIGN_KEY) {
```

Comment:
This is very deeply nested which makes it hard to read. Could we switch to [prefer early return](https://github.com/Shopify/web-configs/blob/main/packages/eslint-plugin/docs/rules/prefer-early-return.md) style here (which is our generally preferred style of coding)?


For example:

```cpp
if (cond->type != ConstraintType::FOREIGN_KEY) {
    continue;
}
if (fk.info.type != ForeignKeyType::FK_TYPE_FOREIGN_KEY_TABLE) {
    continue;
}
...

```